Source/santad/SNTApplication.m

```diff
@@ -61,19 +60,12 @@ - (instancetype)init {
   if (self) {
     SNTConfigurator *configurator = [SNTConfigurator configurator];
 
-    // Choose an event logger.
-    // Locate and connect to driver / SystemExtension
-    if ([configurator enableSystemExtension]) {
```

Comment:
Might as well remove this key from SNTConfigurator.{h,m} and any documentation too

Reply:
Done. Surprisingly, I don't see the EnableSystemExtension key itself mentioned anywhere in the documentation.